creativecloud/features/genfill/genfill-interactive.css

```diff
@@ -16,6 +15,10 @@
   display: none;
 }
 
+.interactive-marquee .genfill-link {
+  cursor: pointer;
```

Comment:
This property can be moved in .genfill .media a {} as it is the same element, no need for another selector.

Reply:
Adding to .genfill .media adds the hand cursor to the enticement too. Hence added it to genfill-link so that it only show up for images

Reply:
.genfill .media a{} applies the css to a tag (not media div), so enticement won't be affected.

Reply:
done